File: android/DFM/src/main/kotlin/com/darakeon/dfm/api/entities/extract/Move.kt
package com.darakeon.dfm.api.entities.extract

import com.darakeon.dfm.api.entities.Date

data class Move(
	val description: String,
	val year: Int,
	val month: Int,
	val day: Int,
	val total: Double,
	var checked: Boolean,
	val id: Int
) {
	val date: Date
		get() = Date(year, month, day)
}

File: site/Entities/Move.cs
using System;
using System.Collections.Generic;
using DFM.Entities.Bases;
using DFM.Entities.Enums;

namespace DFM.Entities
{
	public partial class Move : IMove
	{
		public Move()
		{
			init();
		}

		public virtual Int64 ID { get; set; }
		public virtual Byte[] ExternalId { get; set; }

		public virtual String Description { get; set; }
		public virtual MoveNature Nature { get; set; }
		public virtual Int32 ValueCents { get; set; }

		public virtual Int16 Day { get; set; }
		public virtual Int16 Month { get; set; }
		public virtual Int16 Year { get; set; }

		public virtual Boolean CheckedIn { get; set; }
		public virtual Boolean CheckedOut { get; set; }

		public virtual Category Category { get; set; }
		public virtual Schedule Schedule { get; set; }

		public virtual IList<Detail> DetailList { get; set; }

		public virtual Account In { get; set; }
		public virtual Account Out { get; set; }
	}
}
